datagrepper/app.py

```diff
@@ -131,74 +134,35 @@ def remove_session(exc):
     dm.session.remove()
 
 
-def modify_rst(rst):
-    """Downgrade some of our rst directives if docutils is too old."""
-
-    try:
-        # The rst features we need were introduced in this version
-        minimum = [0, 9]
-        version = map(int, docutils.__version__.split("."))
-
-        # If we're at or later than that version, no need to downgrade
-        if version >= minimum:
-            return rst
-    except Exception:
-        # If there was some error parsing or comparing versions, run the
-        # substitutions just to be safe.
-        pass
-
-    # Otherwise, make code-blocks into just literal blocks.
-    substitutions = {
-        ".. code-block:: javascript": "::",
-    }
-    for old, new in substitutions.items():
-        rst = rst.replace(old, new)
-
-    return rst
-
-
-def modify_html(html):
-    """Perform style substitutions where docutils doesn't do what we want."""
-
-    substitutions = {
-        '<tt class="docutils literal">': "<code>",
-        "</tt>": "</code>",
-    }
-    for old, new in substitutions.items():
-        html = html.replace(old, new)
-
-    return html
-
-
 def preload_docs(endpoint):
     """Utility to load an RST file and turn it into fancy HTML."""
 
     here = os.path.dirname(os.path.abspath(__file__))
     default = os.path.join(here, "docs")
     directory = app.config.get("DATAGREPPER_DOC_PATH", default)
-    fname = os.path.join(directory, endpoint + ".rst")
+    args = " -b html -c %s %s %s/_build/html/" % (default, default, default)
```

Comment:
Since python 3.6, the recommended way to do string formatting would be:
```python
args = f" -b html -c {default} {default} {default}/_build/html/"
```
but since the string is split just below, I would just do something like:
```python
args = ["-b", "html", "-c", default, default, f"{default}/_build/html/"]
```

But then it looks like the last list element should be `htmldocs`, if it was defined before.